### ISSUE TITLE
libXpm & xhost: gettext-tiny is needed

### ISSUE THREAD

**Issue by wael444** [ehawkvu/kiss-xorg#104]
```
make[2]: Leaving directory '/home/wael/.cache/kiss/proc/31085/build/libXpm/src'
Making all in sxpm
make[2]: Entering directory '/home/wael/.cache/kiss/proc/31085/build/libXpm/sxpm'
  CC       sxpm.o
  GEN      sxpm.po
/bin/sh: xgettext: not found
make[2]: *** [Makefile:678: sxpm.po] Error 127
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/wael/.cache/kiss/proc/31085/build/libXpm/sxpm'
make[1]: *** [Makefile:485: all-recursive] Error 1
make[1]: Leaving directory '/home/wael/.cache/kiss/proc/31085/build/libXpm'
make: *** [Makefile:394: all] Error 2
-> libXpm Build failed
```

**Comment by wael444** [comment]
bruh i typed enter on accident

**Comment by ehawkvu** [comment]
I can't reproduce.

```
[ethan ethan] % command -v xgettext
[ethan ethan] % echo $?
1
```

I can build libXaw just fine.

**Comment by wael444** [comment]
I mean libXpm oops

**Comment by ehawkvu** [comment]
You're right. I'd really hate to have anything in `xorg/` depend on gettext so I'm going to see if there's an easy way to fix it.

**Comment by ehawkvu** [comment]
Should be fixed. Let me know if it builds for you.

**Comment by wael444** [comment]
https://termbin.com/wwri6
nope;

**Comment by wael444** [comment]
wait thatx libXpm why do i keep confusing it

**Comment by wael444** [comment]
what that is libXpm not libXpw wtf

**Comment by wael444** [comment]
```
make[2]: Leaving directory '/tmp/kiss/8391/build/xhost/man'
make[2]: Entering directory '/tmp/kiss/8391/build/xhost'
  CC       xhost.o
  GEN      xhost.po
/bin/sh: xgettext: not found
make[2]: *** [Makefile:942: xhost.po] Error 127
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/kiss/8391/build/xhost'
make[1]: *** [Makefile:499: all-recursive] Error 1
make[1]: Leaving directory '/tmp/kiss/8391/build/xhost'
make: *** [Makefile:365: all] Error 2
-> xhost Build failed
```

**Comment by ehawkvu** [comment]
xhost is fixed as of: c72a644b40a71eadfe869f614d3ab7c44adc8512

libXpm is fixed as of: 27d526f6a9a2dce1b78dcc1ea7473eef94fe1a06

**Comment by wael444** [comment]
thank you.!N!>!
